torchinductor/decomposition.py

```diff
@@ -236,3 +238,73 @@ def cudnn_batch_norm(
     null = torch.tensor([], device=input.device)
     null_uint8 = torch.tensor([], device=input.device, dtype=torch.uint8)
     return (result, null, null, null_uint8)
+
+
+@register_decomposition(aten.frac.default, decompositions)
+def frac(input: Tensor) -> Tensor:
+    return input - torch.trunc(input)
+
+
+@register_decomposition(aten.celu.default, decompositions)
+def celu(input: Tensor, alpha: float = 1.0) -> Tensor:
+    inv_alpha = 1.0 / alpha
+    return aten.elu(input, alpha, 1.0, inv_alpha)
+
+
+@register_decomposition(aten.mish.default, decompositions)
+# @pw_cast_for_opmath
```

Comment:
Can we just load these in from torch? Not a big deal, but since you're commenting this stuff out maybe might as well?

Reply:
oh, nvm, didn't realize those hadn't landed yet.